schemas/iso19139/src/main/plugin/iso19139/layout/config-editor.xml

```diff
@@ -725,7 +734,7 @@
                     <gmd:referenceSystemIdentifier>
                       <gmd:RS_Identifier>
                         <gmd:code>
-                          <gco:CharacterString>http://www.opengis.net/def/crs/EPSG/0/4936</gco:CharacterString>
+                          <gmx:Anchor xlink:href="http://www.opengis.net/def/crs/EPSG/0/4936">EPSG:4936</gmx:Anchor>
```

Comment:
does it make sense to introduce a thesaurus for these references, a user can then easily add values to the thesaurus if wants to add (national) projections